openshift/4.0/pipelines/Jenkinsfile.cicd

```diff
@@ -117,6 +121,20 @@ pipeline {
       }
     }
 
+    stage('Approval Prompt') {
```

Comment:
Is there a way to only show a stage if it's configured to be used?  I think this stage would show up in the UI as completed even if it wasn't required.

Reply:
Don't know. In any case it doesn't hurt functionality